pkg/controller/certificaterequest/test_helpers.go

```diff
@@ -58,22 +60,46 @@ var certRequest = &certmanv1alpha1.CertificateRequest{
 			"api.gibberish.goes.here",
 		},
 		Email:             "devnull@donot.route",
-		ReissueBeforeDays: 10000,
+		ReissueBeforeDays: 10,
 	},
+	Status: certmanv1alpha1.CertificateRequestStatus{},
 }
 
-var certRequestAWSPlatformSecrets = &certmanv1alpha1.AWSPlatformSecrets{
-	Credentials: v1.LocalObjectReference{
-		Name: "aws",
+var emptyCertSecret = &corev1.Secret{
+	ObjectMeta: metav1.ObjectMeta{
+		Namespace: testHiveNamespace,
+		Name:      testHiveSecretName,
 	},
-	Region: "not-relevant",
 }
 
-var certSecret = &v1.Secret{
+var validCertSecret = &corev1.Secret{
 	ObjectMeta: metav1.ObjectMeta{
 		Namespace: testHiveNamespace,
 		Name:      testHiveSecretName,
 	},
+	Data: map[string][]byte{
+		// this is an absolutely garbage self-signed cert. it should not be used for
+		// anything ever
+		corev1.TLSCertKey: []byte(`-----BEGIN CERTIFICATE-----
```

Comment:
I'm wondering if it would make better sense to store this as a test asset instead (for easier maintenance and to make the issued dates easier to see).